### ISSUE TITLE
bug: installation error on networkx 

### ISSUE THREAD

**Issue by sjsrey** [spatialucr/geosnap#64]
Trying to install on the ucr cluster I'm hitting:
```
Processing dependencies for osnap==0.1.0
error: networkx 2.2 is installed but networkx<2.0.0 is required by {'region'}
```
Will investigate. Wanted to flag it here in case other had hit this?

**Comment by knaaptime** [comment]
Yeah I get this error on every install, but nothing is broken afaict. 

1) not sure why region is pinned to the older version. The solution might me to update that dependency in region/spopt 

2) we’re defining networkx as a dependency for onsap, so it’s getting installed before region, then we get the warning once the installer gets to region. We don’t really need networkx outside of region, so we could remove this dependency and just allow installation of the version that region is pinned to?

**Comment by knaaptime** [comment]
I think this is resolved by #72
